editor/store/reducer.js

```diff
@@ -584,6 +593,18 @@ export function isTyping( state = false, action ) {
 	return state;
 }
 
+export function currentlyAutosaving( state = false, action ) {
```

Comment:
Noting some naming inconsistencies: `saving` vs. `currentlyAutosaving` (`autosaving`?). Maybe fair because this one reflects a boolean value, whereas the other is a more complex object shape. But then for boolean values we also have `isTyping` vs. `currentlyAutosaving` (`isAutosaving`?).

Reply:
renamed to `isAutosaving` in https://github.com/WordPress/gutenberg/pull/6257/commits/5ee1caea9889e2134979e1b4ac05488cc82bb15b